### ISSUE TITLE
React.createClass is deprecated and will be removed in version 16

### ISSUE THREAD

**Issue by pyramation** [SamyPesse/react-mathjax#6]
As the warning says when using this library:

```
Warning: MathJaxNode: React.createClass is deprecated and will be removed in version 16. Use plain JavaScript classes instead. If you're not yet ready to migrate, create-react-class is available on npm as a drop-in replacement.
```

**Comment by pyramation** [comment]
when this pull is sorted out (https://github.com/SamyPesse/react-mathjax/pull/8) this issue can be closed

**Comment by SamyPesse** [comment]
Fixed in `1.0.0`
